src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/SeleniumTest.kt

```diff
@@ -33,6 +35,7 @@ abstract class SeleniumTest : CustomConfigTests()
     fun setup(hoverfly: Hoverfly)
     {
         hoverfly.simulate(SimulationSource.defaultPath("github-oauth2-login.json"))
+        localConfigs().logLevel(LogLevel.FATAL)
```

Comment:
Just trying to cut down logging verbosity here?

Reply:
yeah hoverfly is annoying. but it didn't work. will remove!